modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -16,12 +16,16 @@
 """
 This module contains basic ignite test.
 """
+import os
 from time import monotonic
 
-from ducktape.tests.test import Test
+from ducktape.utils.local_filesystem_utils import mkdir_p
+from ducktape.tests.test import Test, TestContext
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.utils.ignite_persistence import PersistenceAware, IgnitePersistenceAware
 
 
-# pylint: disable=W0223
+# pylint: disable=C0103,W0223,W0703
```

Comment:
C0103 "Used when a name doesn't doesn't fit the naming convention associated to its type". Why do you disable it?

Reply:
fixed